### DISCUSSION TITLE
How to integrate Phantom into connectKit

### DISCUSSION THREAD

**Discussion by almatutor** [family/connectkit#368] [Q&A] [answered]
I want to integrate Phantom wallet into ConnectKit, but not sure how to do it.
Is there anyone who can guide me how to do?

**Comment by lochie** [comment, marked as answer]
Phantom should automatically appear if the user has it installed, but if you wish to display it even if it's not installed you can set that up by following the following documentation:
https://docs.family.co/connectkit/wallets

It's worth noting that this is a beta feature and the API is subject to change.
